src/rebuild/srv.c

```diff
@@ -530,6 +530,10 @@ ds_rebuild_query(uuid_t pool_uuid, struct daos_rebuild_status *status)
 		}
 	} else {
 		memcpy(status, &rgt->rgt_status, sizeof(*status));
+		if (rgt->rgt_opc == RB_OP_RECLAIM)
+			status->rs_state = DRS_COMPLETED;
+		else
+			status->rs_state = DRS_IN_PROGRESS;
```

Comment:
just confirm two cents
1, if rgt_opc == RB_OP_FAIL_RECLAIM, need it set as COMPLETED also?
2. for other rgt_opc, why set status as IN_PROGRESS? is it possible with rgt != NULL and rgt->rgt_status == DRS_COMPLETED?

Reply:
1. Because exclude might retry if it fail, so it will ignore FAIL_RECLAIM for the moment.
2. ah, you are right. hmm, though rgt->status = completed will be removed from rgt list immediately, it won't "impact" the final result,  though better fix it. thanks!